frontend/src/hooks/useSubFilters.js

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
This really serves as a "filter" for allowed filters. Should we call is something like 'useAllowedFilters'.

Reply:
sure if you want